.travis.yml

```diff
@@ -1,17 +1,14 @@
 language: node_js
 sudo: false
 node_js:
-  - "11.4.0"
+  - '11.4.0'
 
 cache: yarn
 
 install:
   - yarn install
 
 script:
-  # CHECK STYLE
-  - yarn run lint
-
```

Comment:
What was the reason for removing this from the Travis build?

Reply:
I don’t really understand, but for some reason there is a conflict on travis and errors on the lint fall

Reply:
![image](https://user-images.githubusercontent.com/12021443/56417551-fcf75a80-629c-11e9-90c1-c549fa02a2db.png)

But Circle CI works

Reply:
We still need to add this back so that we are getting the full picture of this failure.

Reply:
done